dropwizard-client/src/main/java/io/dropwizard/client/ConfiguredCloseableHttpClient.java

```diff
@@ -0,0 +1,91 @@
+package io.dropwizard.client;
+
+import org.apache.http.HttpHost;
+import org.apache.http.HttpRequest;
+import org.apache.http.client.HttpClient;
+import org.apache.http.client.ResponseHandler;
+import org.apache.http.client.config.RequestConfig;
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.client.methods.HttpUriRequest;
+import org.apache.http.conn.ClientConnectionManager;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.params.HttpParams;
+import org.apache.http.protocol.HttpContext;
+
+import java.io.Closeable;
+import java.io.IOException;
+
+public class ConfiguredCloseableHttpClient implements HttpClient, Closeable {
```

Comment:
It's a shame that `CloseableHttpClient` is an abstract class, not an interface.

I'm happy to apply the change you've suggested as there is probably no need to make external clients exposed to this new class. However, I would much prefer to have a uniform interface.